zulip_bots/zulip_bots/bots/susi/susi.py

```diff
@@ -41,8 +41,36 @@ def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
             return
         reply = requests.get("https://api.susi.ai/susi/chat.json", params=dict(q=msg))
         try:
-            answer = reply.json()['answers'][0]['actions'][0]['expression']
-        except Exception:
+            data = reply.json()
+            actions = data['answers'][0]['actions']
+            answer = ""
```

Comment:
sorry for not finding this before, but I see you are using `"" ` now for strings. In other places, I see you arhe using `''` see line 39. This is inconsistent, and I would just put `''`, which is more clean.